--- a/net/minecraft/world/level/levelgen/RandomSource.java
+++ b/net/minecraft/world/level/levelgen/RandomSource.java
@@ -1,25 +1,25 @@
 package net.minecraft.world.level.levelgen;
 
 public interface RandomSource {
-   void m_142714_(long p_158879_);
+   void setSeed(long p_158879_);
 
-   int m_142703_();
+   int nextInt();
 
-   int m_142421_(int p_158878_);
+   int nextInt(int p_158878_);
 
-   long m_142583_();
+   long nextLong();
 
-   boolean m_142706_();
+   boolean nextBoolean();
 
-   float m_142508_();
+   float nextFloat();
 
-   double m_142055_();
+   double nextDouble();
 
-   double m_142173_();
+   double nextGaussian();
 
    default void m_158876_(int p_158877_) {
       for(int i = 0; i < p_158877_; ++i) {
-         this.m_142703_();
+         this.nextInt();
       }
 
    }
