--- a/net/minecraft/world/level/levelgen/structure/NetherFossilFeature.java
+++ b/net/minecraft/world/level/levelgen/structure/NetherFossilFeature.java
@@ -32,8 +32,8 @@
       }
 
       public void m_142742_(RegistryAccess p_162949_, final ChunkGenerator p_162950_, StructureManager p_162951_, ChunkPos p_162952_, Biome p_162953_, RangeDecoratorConfiguration p_162954_, LevelHeightAccessor p_162955_) {
-         int i = p_162952_.m_45604_() + this.f_73564_.m_142421_(16);
-         int j = p_162952_.m_45605_() + this.f_73564_.m_142421_(16);
+         int i = p_162952_.m_45604_() + this.f_73564_.nextInt(16);
+         int j = p_162952_.m_45605_() + this.f_73564_.nextInt(16);
          int k = p_162950_.m_6337_();
          WorldGenerationContext worldgenerationcontext = new WorldGenerationContext() {
             public int m_142200_() {
