--- a/net/minecraft/world/level/block/BaseEntityBlock.java
+++ b/net/minecraft/world/level/block/BaseEntityBlock.java
@@ -33,6 +33,6 @@
 
    @Nullable
    protected static <E extends BlockEntity, A extends BlockEntity> BlockEntityTicker<A> m_152131_(BlockEntityType<A> p_152132_, BlockEntityType<E> p_152133_, BlockEntityTicker<? super E> p_152134_) {
-      return p_152133_ == p_152132_ ? p_152134_ : null;
+      return p_152133_ == p_152132_ ? (BlockEntityTicker<A>)p_152134_ : null;
    }
 }
