--- a/net/minecraft/world/level/lighting/LayerLightEngine.java
+++ b/net/minecraft/world/level/lighting/LayerLightEngine.java
@@ -185,7 +185,7 @@
    }
 
    public String m_6647_(long p_75694_) {
-      return this.f_75632_.m_6172_(p_75694_).makeConcatWithConstants<invokedynamic>(this.f_75632_.m_6172_(p_75694_));
+      return "" + this.f_75632_.m_6172_(p_75694_);
    }
 
    public void m_142201_(BlockPos p_75686_) {
